Simplified/Reader2/Bookmarks/NYPLBookmarkFactory.swift

```diff
@@ -100,47 +113,50 @@ class NYPLBookmarkFactory {
       return nil
     }
 
-    guard motivation.contains("bookmarking") else {
+    guard motivation.contains(annotationType.rawValue) else {
       return nil
     }
 
-    guard let selector = target["selector"] as? [String: AnyObject],
-      let serverCFI = selector["value"] as? String,
-      let body = annotation["body"] as? [String: AnyObject] else {
-        Log.error(#file, "ServerCFI could not be parsed.")
-        return nil
-    }
+    guard
+      let body = annotation[NYPLBookmarkSpec.Body.key] as? [String: AnyObject],
+      let device = body[NYPLBookmarkSpec.Body.Device.key] as? String,
+      let time = body[NYPLBookmarkSpec.Body.Time.key] as? String,
 
-    guard let device = body["http://librarysimplified.org/terms/device"] as? String,
-      let time = body["http://librarysimplified.org/terms/time"] as? String,
+      // TODO: SIMPLY-3644 fix
       let progressWithinChapter = (body["http://librarysimplified.org/terms/progressWithinChapter"] as? NSNumber)?.floatValue,
-      let progressWithinBook = (body["http://librarysimplified.org/terms/progressWithinBook"] as? NSNumber)?.floatValue else {
+      let progressWithinBook = (body["http://librarysimplified.org/terms/progressWithinBook"] as? NSNumber)?.floatValue
```

Comment:
these strings are R1 only format so I left them as is... they will be removed in SIMPLY-3655